.github/workflows/release.yml

```diff
@@ -122,6 +119,35 @@ jobs:
                 "${UPLOAD_URL}?name=$(basename ${FILE})";
             done
           done
+      - name: Create VERSION.txt
+        run: |
+          echo "$VERSION" > VERSION.txt
+      - name: Upload to artifacts
+        uses: actions/upload-artifact@v3
+        with:
+          name: version
+          path: VERSION.txt
```

Comment:
You might consider making this a [job output](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idoutputs) instead of passing around a file for a simple piece of data.

Reply:
@mcdonnnj resolved with [acfccc9](https://github.com/cisagov/con-pca-tasks/pull/8/commits/acfccc9ae4a98f8d4512331dff61a8b0414da49e)